app.css

```diff
@@ -0,0 +1,66 @@
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+body {
+  padding: 0;
+  margin: 0 auto;
+  width: 80%;
+  font-family: Helvetica, Arial, sans-serif;
+  color: #4D4D4D;
+  font-size: 1em;
+}
+header {
+  text-align: right;
+  padding: 0 10px;
+}
+h1 {
+  font-size: 1.7rem;
+}
+h3 {
+  display: inline-block;
+  width: 50%;
+  font-size: 1.5rem;
+  padding-left: 10px;
+}
+p {
+  display: inline-block;
+  width: 50%;
+  font-size: .7rem;
```

Comment:
I recommend prefixing this with a `0`. People often miss the `.` and are like `7 rem`??!?!?